resources/assets/lib/models/chat/channel.ts

```diff
@@ -51,6 +52,17 @@ export default class Channel {
     return this.messages[this.messages.length - 1];
   }
 
+  @computed
+  get lastMessageId() {
```

Comment:
looks like `lastReadId` ended up a bit behind with this as it's only updated before the actual sent message id is set.